### DISCUSSION TITLE
Zoom to box size

### DISCUSSION THREAD

**Discussion by ershuaili** [vagran/dxf-viewer#103] [Q&A] [answered]
I created a container, but the data did not completely fill the container. There is still some space left. How can I make the container completely filled with data?
![image](https://github.com/vagran/dxf-viewer/assets/69345309/1ecd3e9e-7410-4013-a92b-8ce260acd8f4)

**Comment by vagran** [comment, marked as answer]
It is intentional padding, used for automatic view when a scene is loaded. You can call [FitView()](https://github.com/vagran/dxf-viewer/blob/2468af59659d816612f492cf1caceea587531d54/src/DxfViewer.js#L33) with `padding` argument set to `0` like [here](https://github.com/vagran/dxf-viewer/blob/2468af59659d816612f492cf1caceea587531d54/src/DxfViewer.js#L223). Something like:
```js
const bounds = viewer.GetBounds()
const origin = viewer.GetOrigin()
viewer.FitView(bounds.minX - origin.x, bound.maxX - origin.x, bounds.minY - origin.Y, bounds.maxY - origin.Y, 0)
```
